### PR TITLE
164534477 show next years calendar

### PR DESCRIPTION
**Pull request by rckopteri**
# Added
* Transit visualization page: Show next year only on select or if date is past September first

### CODE REVIEW

File: ote/src/cljs/ote/app/controller/transit_visualization.cljs [hunk 1]
```diff
@@ -176,7 +176,8 @@
                             :all-route-changes-chenckbox false
                             :open-sections {:gtfs-package-info false}
                             :scope scope
-                            :service-changes-for-date-loading? true}]
+                            :service-changes-for-date-loading? true
+                            :show-next-year? (or (t/after? (goog.date.DateTime. (js/Date.)) (goog.date.DateTime. (js/Date. 2019 8 1))) false)}]
```
Comment (philosotractor) [inline ote/src/cljs/ote/app/controller/transit_visualization.cljs]: Hard-coded date forgotten here, replace with a relative one.

**Comment by philosotractor** [comment]
In calendar.cljs the code commented before this PR could be removed:
`#_ (when (and prev-week-hash hash (not= hash prev-week-hash) (> day (time/now)))`

File: ote/src/cljs/ote/app/controller/transit_visualization.cljs (hunk 1 above)
Comment (rckopteri) [reply ote/src/cljs/ote/app/controller/transit_visualization.cljs]: uhh. Thanks!

File: ote/src/cljs/ote/app/controller/transit_visualization.cljs [hunk 2]
```diff
@@ -172,11 +172,17 @@
...
+        initial-view-state {:all-route-changes-display? false
                             :all-route-changes-chenckbox false
                             :open-sections {:gtfs-package-info false}
                             :scope scope
-                            :service-changes-for-date-loading? true}]
+                            :service-changes-for-date-loading? true
+                            :show-next-year? (or
+                                               (t/after?
+                                                 (goog.date.DateTime. (js/Date.))
+                                                 (goog.date.DateTime. (js/Date. current-year 8 1)))
```
Comment (philosotractor) [inline ote/src/cljs/ote/app/controller/transit_visualization.cljs]: Maybe add a somekind comment why Aug 1st was hard-coded (out of convenience when to display the following year?)

**Pull request merged**